errai-data-binding/src/main/java/org/jboss/errai/databinding/rebind/DataBindingUtil.java

```diff
@@ -464,8 +461,56 @@ public static boolean isBindableType(final MetaClass type) {
         }
       }
     }
+  }
 
-    return bindableTypes;
+  private static void addConfiguredBindableTypes(final Set<MetaClass> bindableClasses, String value) {
+    final Set<String> patterns = new LinkedHashSet<>();
+    for (final String s : value.split(" ")) {
+      final String singleValue = s.trim();
+      if (singleValue.endsWith("*")) {
+        patterns.add(singleValue);
+      }
+      else {
+        try {
+          bindableClasses.add(MetaClassFactory.get(singleValue));
+        } catch (final Exception e) {
+          throw new RuntimeException("Could not find class defined in ErraiApp.properties as bindable type: " + s);
+        }
+      }
+    }
+    if (!patterns.isEmpty()) {
+      final SimplePackageFilter filter = new SimplePackageFilter(patterns);
+      MetaClassFactory
+        .getAllCachedClasses()
+        .stream()
+        .filter(mc -> filter.apply(mc.getFullyQualifiedName()) && validateWildcard(mc))
```

Comment:
It also difference in this place. validateWildcard(mc) check added to addConfiguredBindableTypes.

Reply:
Okay. I personally would prefer still having a helper method with some kind of delegate to specify additional filtering, but this is not a blocker for me.